datastore/db/migrations/0019_auto_20230815_1426.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.16 on 2023-08-15 14:26
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0018_auto_20230815_1424'),
+    ]
+
+    operations = [
+        migrations.AddConstraint(
+            model_name='funder',
+            constraint=models.UniqueConstraint(fields=('org_id',), name='funder_unique_org_id'),
+        ),
+        migrations.AddConstraint(
+            model_name='recipient',
+            constraint=models.UniqueConstraint(fields=('org_id',), name='recipient_unique_org_id'),
```

Comment:
We haven't made only having one dataset in the Entity tables an intentional decision but I think it's where we were heading so this looks fine.

Reply:
Aha, I understand now your earlier question about historical data for Entities - I guess if they *did* want to keep the old Entities around, we could have a ForeignKey to getter_run like Publisher does. But they already said not to bother so that makes things easy :)

Just so long as we have some way to uniquely refer to an Entity for the API to lookup.

I suppose then, we could in theory expose 30 days Publisher data via the API, with an extra url/query parameter... but given what they've said, I'm guessing the answer is "only show the latest" until if/when a user requests otherwise?